ModelImporter/import_scene.py

```diff
@@ -20,6 +20,8 @@
 from ..utils.utils import scene_to_dict  # noqa pylint: disable=relative-beyond-top-level
 from .SceneNodeData import SceneNodeData  # noqa pylint: disable=relative-beyond-top-level
 from ..utils.io import get_NMS_dir  # noqa pylint: disable=relative-beyond-top-level
+from ..utils import bpyutils
+from ..utils.bpyutils import SceneOp
```

Comment:
we don't need both import statements, just one (either import `bpyutils`, or import the functions you are using from it... Don't need to do both...)